apps/site/test/site_web/controllers/schedule_controller_test.exs

```diff
@@ -204,6 +204,8 @@ defmodule SiteWeb.ScheduleControllerTest do
       refute :schedules in Map.keys(conn.assigns)
     end
 
+    @tag skip:
+           "Skipping this test temporarily. In Summer 2020 the Green Line C branch is undergoing improvements"
     test "assigns schedules, frequency table, and origin for green line", %{conn: conn} do
```

Comment:
Can this test be edited rather than skipped altogether? Maybe using `Green-D` instead?

Reply:
Yes, that makes more sense 👍